modules/apps/27-interchain-accounts/types/router.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	baseapp "github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// MessageRouter ADR 031 request type routing
```

Comment:
```suggestion
// MessageRouter ADR 031 request type routing
// https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-031-msg-service.md
```

I wonder if we should link the ADR since it doesn't refer to ibc-go ADR's